src/Service/RecipesGenerator.php

```diff
@@ -19,6 +19,8 @@ public function getGeneratedRecipesId(ArrayCollection $selectedTags)
         if (count($selectedTags) != 0) {
             $selectedTags = iterator_to_array($selectedTags);
 
+            $dayCount = 7;
+
             if (in_array('Meat', $selectedTags) == false && in_array('Vegetarian', $selectedTags)) {
```

Comment:
cia irgi hardkodintos reiksmes, bet sikart praleisiu. Realiai reiktu pozymio kad tai vegetarinis patiekalas, nes is stringo to tikrai negalima sprest, nes stringai keiciasi.